test failure in QueryScheduler.spec.js

```
  QueryScheduler
    ✓ executes a function on an interval (2505ms)
    ✓ overwrites interval functions (6510ms)
    ✓ saves queries to file
    ✓ saving a query that already exists updates the query
    ✓ clears and deletes the query if its associated grid was deleted (14287ms)
    1) queries a database and updates a plotly grid on an interval


  26 passing (3m)
  1 pending
  1 failing

  1) QueryScheduler queries a database and updates a plotly grid on an interval:

      AssertionError: expected [ 'Guinea', 'Guinea' ] to deeply equal [ 1 ]
      + expected - actual

       [
      -  "Guinea"
      -  "Guinea"
      +  1
       ]
      
      at Function.assert.deepEqual (node_modules/chai/lib/chai/interface/assert.js:216:32)
      at test/backend/QueryScheduler.spec.js:270:24
      at <anonymous>
      at process._tickDomainCallback (internal/process/next_tick.js:135:7)
```

--- a/test/backend/QueryScheduler.spec.js
+++ b/test/backend/QueryScheduler.spec.js
@@ -276,7 +276,7 @@ describe('QueryScheduler', function() {
             }));
         }
 
-        const refreshInterval = 30; // seconds
+        const refreshInterval = 60; // seconds
         this.timeout(10 * refreshInterval * 1000);
 
         /*

